### PR TITLE
Glossy button.

### PR DESCRIPTION
**Pull request by adsingh14**
Added Glossy button under Special Category.

Issue:

### CODE REVIEW

File: src/components/special/_glossy-btn.less [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    top: 0;
+    height: 25%;
+    background: rgba(white, 0.4);
+    left: 0;
+  }
+  &.white-btn:after {
+    background: rgba(#ddd, 0.4);
+  }
+
+  &:active:after {
```
Comment (shahednasser) [inline src/components/special/_glossy-btn.less:26]: nest this inside `&:active` on line 9 making the necessary adjustments to the selector

**Review by shahednasser — CHANGES_REQUESTED** [review]
I like it but maybe the top part of the button should be reduced? Also, add some transition to the button for the mouse in/out effects to look more seemless. And try to make the box-shadow color lighter.

**Comment by adsingh14** [comment]
> 
> 
> I like it but maybe the top part of the button should be reduced? 
I was thinking to increase the top part. Anyways, I will reduce it.

**Comment by adsingh14** [comment]
#682 Updated.

**Pull request closed without merging**